src/mol-plugin-ui/controls/slider.tsx

```diff
@@ -674,14 +674,12 @@ export class SliderBase extends React.Component<SliderBaseProps, SliderBaseState
         if (val >= max) {
             val = max;
         }
-        /* eslint-disable eqeqeq */
```

Comment:
no more eqeqeq rule in eslint? no needed with TS?

Reply:
`eqeqeq: ["error", "smart"],`

...maybe it's smart enough to not to trigger the warning now :)